src/subdomains/supporting/payin/strategies/send/impl/base/send.strategy.ts

```diff
@@ -55,15 +64,22 @@ export abstract class SendStrategy implements OnModuleInit, OnModuleDestroy {
 
   protected abstract getForwardAddress(): BlockchainAddress;
 
-  protected updatePayInWithSendData(
+  protected async updatePayInWithSendData(
     payIn: CryptoInput,
     type: SendType,
     outTxId: string,
     feeAmount: number = null,
-  ): CryptoInput | null {
+  ): Promise<CryptoInput | null> {
+    const feeAsset = await this.assetService.getNativeAsset(payIn.asset.blockchain);
+    const feeAmountChf = feeAmount
+      ? await this.pricingService
+          .getPrice(feeAsset, this.chf, false)
+          .then((p) => p.convert(feeAmount, Config.defaultVolumeDecimal))
```

Comment:
Can we allow expired prices (`.getPrice(feeAsset, this.chf, true)`) for CHF amounts everywhere?